produce_snp_matrix.py

```diff
@@ -1,46 +1,86 @@
 from glob import glob
-import subprocess
+import json
 import argparse
 import utils
 import os
 
-def snps(input_path, output_path):
+def download_test_case(case, download_path):
+    with open('./test.json') as f:
+        x =json.load(f)
+
+        try:
+            y = x[case]
+        except:
```

Comment:
This maybe seems a little sketchy. It would catch any exceptions and print the statement bellow. 

Is `x` a dict? You could limit the scope to `except KeyError:`